apps/base-docs/tutorials/docs/5_cross-chain-with-layerzero.md

```diff
@@ -184,13 +184,13 @@ In this tutorial, you will be implementing the [OApp](https://docs.layerzero.net
 
 :::info
 
-An extension of the [OApp](https://docs.layerzero.network/contracts/oapp) contract standard known as [OFT](https://docs.layerzero.network/contracts/oft) is also available for supporting omnichain fungible token transfers.
+An extension of the [OApp](https://docs.layerzero.network/contracts/oapp) contract standard known as [OFT](https://docs.layerzero.network/v2/developers/evm/oapp/overview) is also available for supporting omnichain fungible token transfers.
 
```

Comment:
you're using an OApp href for the OFT link. let's use these:
OFT: https://docs.layerzero.network/v2/developers/evm/oft/quickstart
OApp: https://docs.layerzero.network/v2/developers/evm/oapp/overview

thanks!

Reply:
I understand my mistake. I'll create a new one and mention you.

Reply:
https://github.com/base-org/web/pull/1366

@brendan-defi